test/unit/account_test.rb

```diff
@@ -835,4 +835,14 @@ def test_smart_destroy_master
     assert master_account.reload
     refute master_account.reload.scheduled_for_deletion?
   end
+
+  def test_api_docs_services_without_service
```

Comment:
I would not create this method in Account and only test the scope in `ApiDocs::Service`

It would be lighter